tests/qam-updinstall/update_install.pm

```diff
@@ -168,13 +179,15 @@ sub run {
     }
     record_info('Modules', "@modules");
 
-    # Patch the SUT to a released state;
-    fully_patch_system;
+    # Patch the SUT to a released state and reboot if reboot is needed;
+    reboot_and_login if fully_patch_system == 102;
 
     set_var('MAINT_TEST_REPO', $repos);
     my $repos_count = add_test_repositories;
     record_info('Repos', script_output('zypper lr -u'));
 
+    my $rollback_number = script_output('snapper create --description "Pre-patch" -p');
```

Comment:
One minor comment, could you add a `record_info` here to record creating a new snapshot here?

Reply:
done